zookeeper-client/zookeeper-client-c/pom.xml

```diff
@@ -97,7 +58,7 @@
           </execution>
           <execution>
             <id>configure</id>
-            <phase>test-compile</phase>
+            <phase>process-sources</phase>
```

Comment:
This is where --enable-gcov is passed, the phase which this runs probably surfaced the problem for Patrick? If someone doesn't have the library it will always fail, with ant only if they run a c-coverege report.
Thanks @symat for finding this!